src/main/java/com/ning/http/client/generators/InputStreamBlockingBodyGenerator.java

```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package com.ning.http.client.generators;
-
-import java.io.InputStream;
-
-/**
- * Same as {@link InputStreamBodyGenerator} excepts it indicates that the writing should block because the data is large.
- */
-public class InputStreamBlockingBodyGenerator extends InputStreamBodyGenerator {
```

Comment:
We created a hot fix with this at some point for 3.x. Scan our code to make sure we are not using it and replace it if we are (this fix needs to go into 3.x anyways).

Reply:
This class is used in mule-3.x. Should we use it or should we change it by the feedable strategy instead?

Reply:
It needs to be removed in favor of the feedable strategy.